examples/resources/plugins/teleport-pagerduty.toml

```diff
@@ -1,26 +1,25 @@
 # example teleport-pagerduty configuration TOML file
 [teleport]
-auth-server = "0.0.0.0:3025"  # Auth GRPC API address
-client-key = "/var/lib/teleport/plugins/pagerduty/auth.key" # Teleport GRPC client secret key
-client-crt = "/var/lib/teleport/plugins/pagerduty/auth.crt" # Teleport GRPC client certificate
-root-cas = "/var/lib/teleport/plugins/pagerduty/auth.cas"   # Teleport cluster CA certs
+auth_server = "example.com:3025"                            # Teleport Auth Server GRPC API address
+client_key = "/var/lib/teleport/plugins/pagerduty/auth.key" # Teleport GRPC client secret key
+client_crt = "/var/lib/teleport/plugins/pagerduty/auth.crt" # Teleport GRPC client certificate
+root_cas = "/var/lib/teleport/plugins/pagerduty/auth.cas"   # Teleport cluster CA certs
 
 [pagerduty]
-api-key = "key"               # PagerDuty API Key
-user-email = "me@example.com" # PagerDuty bot user email (Could be admin email)
-service-id = "PIJ90N7"        # PagerDuty service id
+api_key = "key"               # PagerDuty API Key
+user_email = "me@example.com" # PagerDuty bot user email (Could be admin email)
+service_id = "PIJ90N7"        # PagerDuty service id
 
 [http]
-listen = "0.0.0.0:8081"          # PagerDuty webhook listener
-base-url = "https://teleport-pagerduty.infra.yourcorp.com" # The public address of the teleport-pagerduty webhook listener.
-# host = "example.com"    # Host name by which bot is accessible
-# https-key-file = "/var/lib/teleport/plugins/pagerduty/server.key"  # TLS private key
-# https-cert-file = "/var/lib/teleport/plugins/pagerduty/server.crt" # TLS certificate
+public_addr = "example.com" # URL on which callback server is accessible externally, e.g. [https://]teleport-pagerduty.example.com
```

Comment:
Address at the end should match the address at the beginning

i.e. `public_addr = "teleport-pagerduty.example.com"`